### PR TITLE
refactored:video library

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**

### CODE REVIEW

**Review by arpit01923 — APPROVED** [review]
- code looks clean
- good to merge

File: src/PortalComponent/modal.module.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
   background: rgba(0, 0, 0, 0.3);
   display: flex;
   justify-content: center;
-  align-items: center;
+  /* align-items: center; */
```
Comment (ashishkatiyar2404) [inline src/PortalComponent/modal.module.css:6]: you can remove comments

File: src/component/LayoutComponent/header/Header.module.css [hunk 2]
```diff
@@ -48,7 +48,17 @@
...
+.mobile_search {
+  width: 100%;
 
+  padding: 1.5rem 3rem;
+  height: 100%;
+  outline: none;
+  border: 0px;
+  background-color: whitesmoke;
+  /* box-shadow: rgba(0, 0, 0, 0.05) 0px 6px 24px 0px,
+    rgba(0, 0, 0, 0.08) 0px 0px 0px 1px; */
```
Comment (ashishkatiyar2404) [inline src/component/LayoutComponent/header/Header.module.css:60]: here also

File: src/component/layoutComponent/header/Header.module.css [hunk 3]
```diff
@@ -48,7 +48,17 @@
...
+.mobile_search {
+  width: 100%;
 
+  padding: 1.5rem 3rem;
+  height: 100%;
+  outline: none;
+  border: 0px;
+  background-color: whitesmoke;
+  /* box-shadow: rgba(0, 0, 0, 0.05) 0px 6px 24px 0px,
+    rgba(0, 0, 0, 0.08) 0px 0px 0px 1px; */
```
Comment (ashishkatiyar2404) [inline src/component/layoutComponent/header/Header.module.css:60]: you can also remove comments form here

**Review by manojncsoni007 — APPROVED** [review]
- Everything is well and good
Go ahead, Good to merge

File: src/customHooks/useDocumentTitle.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ const useDocumentTitle = () => {
 
   useEffect(() => {
     window.document.title = `FrontEnd|${title_}`;
+    window.scrollTo({ top: 0, behavior: "auto" });
   }, [title_]);
```
Comment (ashishkatiyar2404) [inline src/customHooks/useDocumentTitle.js:12]: you can use `title` name simply if there are more words like `titlePrice or titleXYZ` thn you can use `title_XYZ`
what u have used its also ok someone suggested me for better

**Review by ashishkatiyar2404 — APPROVED** [review]
Looks good to go. Suggested few changes pls look into.

**Pull request merged**